### PR TITLE
Updated foreman_discovery to 4.1.2 (DEB)

### PR DESCRIPTION
**Pull request by lzap**
Ditto, 1.10 please, thanks!

### CODE REVIEW

**Comment by domcleal** [comment]
The failure seems to be a problem on the build servers caused by the bad package name in #862 being cached somewhere, I'm trying to figure out where it's coming from.

**Comment by domcleal** [comment]
No, I was reading the wrong log from Jenkins.  The problem is the branch name - please change "dev/" to "deb/" by opening a new PR.

File: plugins/ruby-foreman-discovery/debian/changelog [hunk 1]
```diff
@@ -1,3 +1,9 @@
+ruby-foreman-discovery (4.1.2) stable; urgency=low
```
Comment (domcleal) [inline plugins/ruby-foreman-discovery/debian/changelog]: Note that you'll need to append -1 to the version number as per #867 until it's merged.

**Comment by mmoll** [comment]
merged with the version fixup after testing as 4efefc7 for develop and 26ef673 for 1.10, dík @lzap!

**Pull request closed without merging**